engine/dlib/src/dlib/ssdp.cpp

```diff
@@ -769,7 +769,7 @@ namespace dmSSDP
         if (sr != dmSocket::RESULT_OK)
         {
             // When returning from sleep mode on iOS socket is in state ECONNABORTED
-            if (sr == dmSocket::RESULT_CONNABORTED || sr == dmSocket::RESULT_NOTCONN)
+            if (sr == dmSocket::RESULT_CONNABORTED || sr == dmSocket::RESULT_NOTCONN || sr == dmSocket::RESULT_NETDOWN)
```

Comment:
An issue in debug mode where, if the app comes back after sleep, the connection is lost, but we just continue the loop checking for more content (-> hang)